styles.css

```diff
@@ -198,3 +200,532 @@ a {
   color: unset;
   cursor: pointer;
 }
+
+#action-btn-cont {
+  position: absolute;
+  top: 665px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 327px;
+  height: 48px;
+}
+
+#action-btn {
+  width: 40px;
+  height: 40px;
+  background-image: url("images/Disabled.png");
+  background-size: cover;
+}
+
+/* #endregion */
+
+/* #region Works Section */
+
+#works {
+  display: flex;
+  position: absolute;
+  top: 708px;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 3283px;
+}
```

Comment:
- [ ] This section is the main container of the work `section`. The main container should have `display: grid`. This means all cards should be arranged using `grid` not `flexbox`. Kindly fix this issue.

Reply:
I will share the instruction screenshot. So you can see that is not mandatory. This is my works section container that's right. But the instruction tells the main container must have grid attribute that contains all of your cards. So my cards container has grid attribute. Please take a look of my codes again. By the way, thanks for your time for all of your revisions.
![Uploading Ekran görüntüsü 2023-06-03 183031.png…]()

Reply:
Hi, @s2jack 👋 
Sorry for the misconception. I can see you are right.
Happy Coding 😄